plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -0,0 +1,172 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.eclipse.jface.wizard.IWizardPage;
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.ModifyEvent;
+import org.eclipse.swt.events.ModifyListener;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.DirectoryDialog;
+import org.eclipse.swt.widgets.Group;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+
+/**
+ * UI to collect all information necessary to create a new App Engine Standard Java Eclipse project.
+ */
+public class AppEngineStandardWizardPage extends WizardPage implements IWizardPage {
```

Comment:
org.eclipse.ui.workbench and some rap thing.

What feature do I need to add to the target platform to pull in org.eclipse.ui.ide? More importantly, how is one supposed to figure out what feature contains a certain package? Is there somewhere a list of feature groups and the packages they contain or vice versa?